examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb

```diff
@@ -1877,7 +1881,7 @@
     "\n",
     "def edit_wireframe_UVSphere(edit_template):\n",
     "    # @markdown Number of (line) rings. Must be larger or equal to 2 and multiple of 2.\n",
-    "    num_rings = 16  # @param {type:\"slider\", min:2, max:64, step:2}\n",
+    "    num_rings = 16  # @param {type:\"slider\", min:2, max:64, step:2} # noqa: F841\n",
     "    # @markdown Number of (line) segments. Must be larger or equal to 4 and multiple of 4.\n",
     "    num_segments = 40  # @param {type:\"slider\", min:4, max:64, step:4}\n",
     "    edit_template.num_segments = num_segments\n",
```

Comment:
Add `edit_template.num_rings = num_rings`.

Reply:
Correct me if wrong, @jturner65 🙂

Reply:
yes that is correct

Reply:
So get rid of that  # noqa: F841
and add 
`edit_template.num_rings = num_rings`